### DISCUSSION TITLE
Configuring Nginx and PHP-FPM (and Nette) – Blog

### DISCUSSION THREAD

**Discussion by giscus** [jiripudil/jiripudil.cz#223] [Blog discussions]
# Configuring Nginx and PHP-FPM (and Nette) – Blog

A full-stack web developer from Brno

https://jiripudil.cz/blog/configuring-nginx-php-fpm-nette

**Comment by aleroma** [comment]
Is this relevant in 2023?

**Reply by jiripudil** [reply]
Hi, as far as I know, yes; not much has changed since then about the configuration. There's also a tutorial on Digital Ocean that seems to be more up-to-date, and it says basically the same: https://www.digitalocean.com/community/tutorials/php-fpm-nginx
